math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -2530,8 +2530,8 @@ RVec<T> Where(const RVec<int>& c, const RVec<T>& v1, T v2)
 /// if_c_v1_else_v2
 /// // (ROOT::VecOps::RVec<double>) { 1.0000000, 4.0000000, 4.0000000 }
 /// ~~~
-template <typename T>
-RVec<T> Where(const RVec<int>& c, T v1, const RVec<T>& v2)
+template <typename T, typename U, typename = std::enable_if<std::is_convertible<T, U>::value, void>>
```

Comment:
```suggestion
template <typename T, typename U, typename = typename std::enable_if<std::is_convertible<T, U>::value>::type>
```
or what am I missing? (And yes we need concepts.)

Reply:
ah wow, of course I meant `std::enable_if_t`, thanks!